usr/src/test/zfs-tests/tests/functional/cli_root/zfs_program/zfs_program_json.ksh

```diff
@@ -0,0 +1,132 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy is of the CDDL is also available via the Internet
+# at http://www.illumos.org/license/CDDL.
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2018 Datto Inc.
+#
+
+. $STF_SUITE/usr/src/common/zfs/libtest.shlib
```

Comment:
why `$STF_SUITE/usr/src/common/zfs/libtest.shlib`?
and not `. $STF_SUITE/include/libtest.shlib`?

I saw your makefiles doing that too. Could you maybe change the Makefiles and the include in that file to be consistent with the rest of the stuff under `cli_root`?

Reply:
fixed this in the test but not sure which Makefile you were talking about

Reply:
I'm sorry, my mistake. I meant the new `cleanup.ksh` and `setup.ksh` have the same issue (should be `. $STF_SUITE/include/libtest.shlib` instead of `$STF_SUITE/usr/src/common/zfs/libtest.shlib`) - not Makefiles.

Reply:
got it, I've made the change in the latest update